monitors/test_settings.py

```diff
@@ -11,10 +11,11 @@
 DATA_LOC = os.path.join('data', 'cycle_%s')
 SUMMARY_LOC = os.path.join('logs', 'journal', 'summary.txt')
 LAST_RUN_LOC = os.path.join('logs', 'lastrun.txt')
-LOG_FILE = os.path.join(get_project_root(), 'logs', 'monitor.log')
-INSTRUMENTS = [{'name': 'WISH', 'use_nexus': True},
-               {'name': 'GEM', 'use_nexus': True},
-               {'name': 'OSIRIS', 'use_nexus': True},
-               {'name': 'POLARIS', 'use_nexus': True},
-               {'name': 'MUSR', 'use_nexus': True},
-               {'name': 'POLREF', 'use_nexus': True}]
+EORM_LOG_FILE = os.path.join(get_project_root(), 'logs', 'end_of_run_monitor.log')
+ICAT_MON_LOG_FILE = os.path.join(get_project_root(), 'logs', 'icat_monitor.log')
+INSTRUMENTS = [{'name': 'WISH', 'file_prefix': 'WISH', 'use_nexus': True},
```

Comment:
As I mentioned in the PR, MERLIN would be an example where the file prefix is MER. Just thinking about future expansion. For now it would be fine to keep the existing set up.

Reply:
You're right apologies I missed this in the PR description